lumenlearning/IMathAS
# Modifypostfeild

# What

Changed Modify Posts page to activeform
# Test

1)Switch to this branch on local `Modifypostfeild`
2)Then Make a forum (Or use one that already exists)
3) Add a new thread to the forum
4)Click on the modify post button 
![screen shot 2016-06-08 at 10 34 00 am](https://cloud.githubusercontent.com/assets/6364181/15904106/8bf73dde-2d64-11e6-9889-4712fcb6eaed.png)

5) then on the modify page empty the subject feild 
    It should highlight and say  `Subject cannot be empty`
6) Make a change and save it. 
7)Make sure the changes show
